### ISSUE TITLE
Doesn't stop when systemctl managed

### ISSUE THREAD

**Issue by pbrisbin** [timhaak/docker-couchpotato#5]
tl;dr: I think the `./start.sh` script needs to use `exec`.

I'm running this image as a systemd unit

```
[Unit]
Description=CouchPotato
Requires=docker.service
After=docker.service

[Service]
TimeoutStartSec=0
Restart=always
ExecStartPre=-/usr/bin/docker stop couchpotato
ExecStartPre=-/usr/bin/docker rm couchpotato
ExecStartPre=/usr/bin/docker pull timhaak/couchpotato
ExecStart=/usr/bin/docker run --name couchpotato \
  --publish 5050:5050 \
  --volume /mnt/media:/media \
  --volume /data/couchpotato:/data \
  --volume /data/couchpotato:/config \
  timhaak/couchpotato

[Install]
WantedBy=multi-user.target
```

When I execute `systemctl stop docker-couchpotato.service` it pauses for a long time, eventually returns, but hasn't stopped the container. I have to find it with `docker ps` and use `docker stop` which is also slow but does stop things.

I'm 99% sure this is because the start script doesn't use `exec` and so the couchpotato process is not PID 1 and doesn't exit in response to the `TERM` systemd is sending.

Your sickbeard image, which does use `exec`, works fine when stopped via `systemctl`.

Tomorrow I should be able to build an image to test things out, but I wanted to capture this while I was thinking of it.

Thanks for all your images by the way. I converted my main media server over to all docker containers tonight and it was dead-simple thanks to your effort.
